hooks/src/index.js

```diff
@@ -206,7 +206,12 @@ function scheduleFlushAfterPaint() {
 if (typeof window !== 'undefined') {
 	afterPaint = (component) => {
 		if (!component._afterPaintQueued && (component._afterPaintQueued = true) && afterPaintEffects.push(component) === 1) {
-			requestAnimationFrame(scheduleFlushAfterPaint);
+			if (options.afterPaint) {
+				options.afterPaint(flushAfterPaintEffects);
```

Comment:
Can we change this to `options.requestAnimationFrame`? It should compress better.